### ISSUE TITLE
AngularJS out of date

### ISSUE THREAD

**Issue by Hoverbear** [scotch/angular-brunch-seed#35]
Currently using Angular JS 1.0.2 when stable is 1.0.3.

**Comment by Hoverbear** [comment]
This should maybe be pulling off CDN now that I think about it. Thoughts?

**Comment by kylefinley** [comment]
I'm undecided on this.

Pros:
- Faster load times if the user has a cached version
- Simpler less files to include

Cons:
- Difficult to debug
- Must be connected to the internet durring development
- Google CDN can sometime be blocked

Also, [angular-seed](https://github.com/angular/angular-seed) is still including the scripts.

Maybe we could we could add commented out lines that would add Angular through the CDN and remove the script files from the build.

**Comment by Hoverbear** [comment]
Maybe use a fallback?

```
<!-- Angular JS off CDN, otherwise fallback. -->
<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.0.2/angular.min.js"></script>
 <script>
    window.angular || document.write('<script src="scripts/vendor/angular-1.0.2.min.js"><\/script>')
 </script>
```

For example

**Comment by derek-adair** [comment]
fallback is the way to go!

**Comment by derek-adair** [comment]
Actually, it might be cool to have brunch swap out the local script for the proper CDN version on production compile.  I have no idea how difficult that would be to accomplish... but that'd be pretty slick!

**Comment by kylecordes** [comment]
I'm not sure if this is maintained much anymore... but if so, relevant to this item Angular is now at 1.2.3.

**Comment by ethanmick** [comment]
`1.3.8` Is the latest version, 1.2.x is now legacy.
